libs/dataset_deployer.py

```diff
@@ -78,14 +78,20 @@ def write_nested_csv(
 
     Args:
         data: list of data to write.
-        key: Stem of file to write
-        output_dir: Output directory to write to.
+        output_path: Path of file to write to.
+        keys_to_skip: Keys to skip.  Keys can be flattened entries or top level keys.
+
     """
+    keys_to_skip = keys_to_skip or []
+
     if not data:
         raise ValueError("Cannot upload a 0 length list.")
-    header = flatten_dict(data[0]).keys()
-    if keys_to_skip:
-        header = [column for column in header if column not in keys_to_skip]
+
+    first_row = data[0]
```

Comment:
Add a comment, maybe
`# Skip top level keys before passing first_row to flatten_dict`